zanata-client-commands/src/main/java/org/zanata/client/commands/PushPullCommand.java

```diff
@@ -89,13 +92,8 @@ public PushPullCommand(O opts) {
 
     protected void confirmWithUser(String message) throws IOException {
         if (getOpts().isInteractiveMode()) {
-            Console console = System.console();
-            if (console == null) {
-                throw new RuntimeException(
-                        "console not available: please run Maven from a console, or use batch mode option (-B)");
-            }
-            console.printf(message + "\nAre you sure (y/n)? ");
-            expectYes(console);
+            console.printf(Question, message + "\nAre you sure (y/n)? ");
```

Comment:
`ConsoleInteractorImpl` doesn't appear to handle the case where `System.console()` is null (when running with `nohup` or something?), so I think this introduces a danger of NPE.